src/main/java/chess/view/OutputView.java

```diff
@@ -0,0 +1,43 @@
+package chess.view;
+
+import chess.domain.Board;
+import chess.domain.Piece;
+import java.util.ArrayList;
+import java.util.List;
+
+public class OutputView {
+
+    public void printStartMessage() {
+        System.out.println("체스 게임을 시작합니다.");
+        System.out.println("게임 시작 : start");
+        System.out.println("게임 종료 : end");
+        System.out.println("게임 이동 : move source위치 target위치 - 예. move b2 b3");
```

Comment:
```suggestion
    public void printStartMessage() {
        System.out.println("체스 게임을 시작합니다.\n");
                        + "게임 시작 : start")
                        + "게임 종료 : end")    
                        + "게임 이동 : move source위치 target위치 - 예. move b2 b3");
```
위의 코드 비교했을 때 System.out.println 반복 호출은 프로그램에 어떤 영향을 미칠까요?
(키워드: synchronized 키워드)